systemd/ocp-clusterid.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -x
+
+export KUBECONFIG="/opt/kubeconfig"
+uuid=$(uuidgen)
+
+retry=0
+max_retry=20
+until `oc get clusterversion > /dev/null 2>&1`
+do
+    [ $retry == $max_retry ] && exit 1
+    sleep 5
+    ((retry++))
+done
```

Comment:
I see this block of code duplicated across various scripts where we are waiting for some resource to become available. I'm not 100% sure if it's possible to do it in SystemD setup but can we move this to a generic method that can handle various resource types? WDYT?

Reply:
Maybe with https://fedoramagazine.org/systemd-template-unit-files/
Or just by using a shared script.

Reply:
sure, I'll extract this into a function that we can re-use across scripts